modules/nf-core/rgi/cardannotation/main.nf

```diff
@@ -0,0 +1,60 @@
+process RGI_CARDANNOTATION {
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/rgi:6.0.3--pyha8f3691_0':
+        'biocontainers/rgi:6.0.3--pyha8f3691_0' }"
+
+    input:
+    path(card)
+
+    output:
+    path("output_dir")  , emit: db
+    env RGI_VERSION     , emit: tool_version
+    env DB_VERSION      , emit: db_version
+    path "versions.yml" , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+
+    """
+    rgi card_annotation -i $card/card.json \\
+    $args
+
+    DB_VERSION=\$(ls card_database_*_all.fasta | sed "s/card_database_v\\([0-9].*[0-9]\\).*/\\1/")
+
+    mkdir output_dir
```

Comment:
Is this because of variability in the database name? Iw ould make the directory name more descriptive e.g. card_database/

Reply:
It might be too risky that the user supplies a folder with that name so that `mkdir` won't work. Maybe `card_database_processed` is good enough?

Reply:
Yeah might be fine

Reply:
Or use a prefix...